charts/yourls/values.yaml

```diff
@@ -477,7 +477,13 @@ ingress:
   ## e.g:
   ## extraHosts:
   ##   - name: yourls.local
-  ##     path: /
+  ##     path: /foo
+  ##     pathType: Prefix # optional
+  ##     backend:         # optional
+  ##       service:
+  ##         name: response-foo
+  ##         port:
+  ##           name: use-annotation
```

Comment:
This backend example is specific to the aws load balancer ingress controller: https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/ingress/annotations/#actions